### PR TITLE
Queries and templates for Drupal Q&A Section and Q&A paragraphs.

### PR DESCRIPTION
**Pull request by beeyayjay**
## Description
Added a query and template for Q&A Section and a query and template for Q&As.

## Testing done
Checked random pages with q&a.

## Definition of done
- [ ] All Q&As are wrapped in a scema.org tag (<div itemscope="" itemtype="http://schema.org/Question">)
- [ ] All Questions are either h2 or h3 and have a name itemprop (<h2 itemprop="name">)
- [ ] All Answers have and 'accepted answer' itemprop (<div itemprop="acceptedAnswer" itemscope="" itemtype="http://schema.org/Answer">)

### CODE REVIEW

File: src/site/stages/build/drupal/graphql/paragraph-fragments/qa.paragraph.graphql.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+/**
+ * A Drupal paragraph for questions and answers. Answers come in the format of WYSIWYG, collapsible panel, and process.
+ *
+ */
+module.exports = `
+	fragment qa on ParagraphQA {
```
Comment (jbalboni) [inline src/site/stages/build/drupal/graphql/paragraph-fragments/qa.paragraph.graphql.js]: Extra indentation here

File: src/site/stages/build/drupal/graphql/paragraph-fragments/qaSection.paragraph.graphql.js [hunk 2]
```diff
@@ -1,47 +1,16 @@
...
-              fieldSteps {
-                processed
-              }
-                }
-              }
-            }
-          }
-        }
+  fragment qaSection on ParagraphQASection {
+  	fieldSectionHeader
```
Comment (jbalboni) [inline src/site/stages/build/drupal/graphql/paragraph-fragments/qaSection.paragraph.graphql.js]: Indentation issue here, too

**Review by jbalboni — APPROVED** [review]
Two minor nitpicks, otherwise LGTM

File: src/site/stages/build/drupal/graphql/paragraph-fragments/qa.paragraph.graphql.js (hunk 1 above)
Comment (beeyayjay) [reply src/site/stages/build/drupal/graphql/paragraph-fragments/qa.paragraph.graphql.js]: Sorry about that (tab instead of spaces). Fixed now.

**Pull request merged**